server/sonar-server/src/main/java/org/sonar/server/test/index/TestIndex.java

```diff
@@ -93,7 +97,7 @@ public TestDoc getByTestUuid(String testUuid) {
       .setQuery(boolQuery().must(matchAllQuery()).filter(termQuery(FIELD_TEST_UUID, testUuid)))
       .get().getHits().getHits();
     if (hits.length > 0) {
-      return Optional.of(new TestDoc(hits[0].sourceAsMap()));
+      return Optional.of(new TestDoc(hits[0].getSourceAsMap()));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "sourceAsMap"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)